privaterelay/views.py

```diff
@@ -59,12 +65,24 @@ def faq(request):
 def profile(request):
     if (not request.user or request.user.is_anonymous):
         return redirect(reverse('fxa_login'))
+    if request.GET.get('fxa_refresh') == 1:
+        social_account = request.user.socialaccount_set.filter(
+            provider='fxa'
+        ).first()
+        _handle_fxa_profile_change(social_account)
     relay_addresses = RelayAddress.objects.filter(user=request.user).order_by(
         '-created_at'
     )
+    domain_addresses = DomainAddress.objects.filter(user=request.user).order_by(
+        '-last_used_at'
```

Comment:
Yeah, I wonder if we should sort relay addresses by `-last_used_at` too.

Reply:
I think actually the opposite. I think both should be `-created_at`. `-last_used_at` I think may change the orders on the dashboard too often?

Reply:
We don't have `DomainAddress.created_at` because we don't know when the user "created" a domain address - so, instead I put `DomainAddress.first_emailed_at`.

Should we punt any decision until we have UX dedicated to it?

Reply:
I see. What about when a user generates the subdomain alias via add-on? That would apply the `created_at` like the RelayAddress would. Maybe we just use both?